dask/array/__init__.py

```diff
@@ -28,3 +28,4 @@
 from ..base import compute
 from .optimization import optimize
 from .creation import arange, linspace, diag
+from .fft import fft, ifft
```

Comment:
This is because the modules name is `fft.py`, renaming suggestions? NumPy uses `fftpack.py`. 

`fft_module.py`? `fft_.py`?

I also forgot to import the other functions.